docs/source/prerequisites/index.rst

```diff
@@ -22,14 +20,17 @@ roles are illustrated as follows:
    The "core" agentless functionality of **cijoe**; run commands and tranfer
    files
 
+.. _sec-prerequisites-initiator:
+
 Initiator
 ---------
 
 The **initiator** is the system where the **cijoe** command-line tool is
-executed, and the **cijoe** scripts are run, requiring :python:`Python <>` along
-with the necessary :python:`Python <>` package dependencies. With the adoption
-of :pep668:`PEP 668 <>` by Linux distributions, providing a virtual environment
-for Python packages is now mandatory rather than a recommended practice.
+invoked, and the **cijoe** scripts are executing, requiring :python:`Python
```

Comment:
no comma after "invoked" and maybe change to "and the **cijoe** scripts are being executed" instead of "are executing"